$NetBSD: patch-configure,v 1.5 2021/01/01 12:46:03 rhialto Exp $

1. Ignore the option --enable-option-checking instead of erroring,
   it's the default anyway.

--- configure.orig	2020-12-23 12:58:34.000000000 +0000
+++ configure
@@ -4173,6 +4173,8 @@ for argcheck in $@
 do
   argvalid=yes
   case $argcheck in #(
+  --enable-option-checking=*)
+    argvalid=yes ;; #(
   --enable-arch=*) :
     argvalid=yes ;; #(
   --enable-make-command*) :
